modules/installation-overview.adoc

```diff
@@ -5,28 +5,37 @@
 
 :_content-type: CONCEPT
 [id="installation-overview_{context}"]
-= {product-title} installation overview
 
-The {product-title} installation program offers you flexibility. You can use the installation program to deploy a cluster on infrastructure that the installation program provisions and the cluster maintains or deploy a cluster on infrastructure that you prepare and maintain.
+The {product-title} installation program offers you flexibility. You can use the program to:
 
-These two basic types of {product-title} clusters are frequently called installer-provisioned infrastructure clusters and user-provisioned infrastructure clusters.
+* To deploy a cluster on provisioned infrastructure and the cluster maintains.
+* Deploy a cluster on infrastructure that you prepare and maintain.
 
-Both types of clusters have the following characteristics:
+These two basic types of {product-title} clusters are frequently called:
 
-* Highly available infrastructure with no single points of failure is available by default
-* Administrators maintain control over what updates are applied and when
+* Installer-provisioned infrastructure clusters.
+* User-provisioned infrastructure clusters.
 
-You use the same installation program to deploy both types of clusters. The main assets generated by the installation program are the Ignition config files for the bootstrap, master, and worker machines. With these three configurations and correctly configured infrastructure, you can start an {product-title} cluster.
+Both cluster types have the following characteristics:
 
-The {product-title} installation program uses a set of targets and dependencies to manage cluster installation. The installation program has a set of targets that it must achieve, and each target has a set of dependencies. Because each target is only concerned with its own dependencies, the installation program can act to achieve multiple targets in parallel. The ultimate target is a running cluster. By meeting dependencies instead of running commands, the installation program is able to recognize and use existing components instead of running the commands to create them again.
+* Highly available infrastructure with no single points of failure is available by default.
+* Administrators maintain control over what updates are applied and when.
 
-The following diagram shows a subset of the installation targets and dependencies:
+[id="about-the-installation-program"]
+== About the installation program
+
+You can use the installation program to deploy both types of clusters. The installation program generates main assets such as Ignition config files for the bootstrap, master, and worker machines. You can start an  {product-title} cluster with these three configurations and correctly configured infrastructure.
```

Comment:
Remove extra space in (after "an"):"...You can start an  {product-title} cluster with..."